testing/integration/container_cmd_test.go

```diff
@@ -327,6 +327,9 @@ func TestContainerCMDEventToStderr(t *testing.T) {
 	agentFixture, err := define.NewFixtureFromLocalBuild(t, define.Version())
 	require.NoError(t, err)
 
+	// We call agentFixture.Prepare to set the workdir
+	require.NoError(t, agentFixture.Prepare(ctx))
```

Comment:
nit: add a little bit of context to the `require` (not mandatory but removes the need to look at the source to see what the test was doing when the unexpected error occurred)
```suggestion
	require.NoError(t, agentFixture.Prepare(ctx), "failed preparing agent fixture")
```